pyuvdata/fhd.py

```diff
@@ -198,7 +241,7 @@ def read_fhd(self, filelist, use_model=False, run_check=True, check_extra=True,
             warnings.warn('Ntimes does not match the number of unique times in the data')
         self.time_array = np.zeros(self.Nblts)
         if self.Ntimes == 1:
-            self.time_array.fill(int_times)
+            self.time_array.fill(int_times[0])
```

Comment:
Was this a change to the baseline info dict?

Reply:
No, it was a bug. This line of code wasn't covered before this PR and when I added the test coverage for this case it errored until I made this change.